OrbitLinuxTracing/include/OrbitLinuxTracing/Events.h

```diff
@@ -44,6 +44,30 @@ class ContextSwitchOut : public ContextSwitch {
       : ContextSwitch(pid, tid, core, timestamp_ns) {}
 };
 
+class SchedulingSlice {
+ public:
+  SchedulingSlice(pid_t pid, pid_t tid, uint16_t core, uint64_t in_timestamp_ns,
+                  uint64_t out_timestamp_ns)
+      : pid_(pid),
+        tid_(tid),
+        core_(core),
+        in_timestamp_ns_(in_timestamp_ns),
+        out_timestamp_ns_(out_timestamp_ns) {}
+
+  pid_t GetPid() const { return pid_; }
```

Comment:
maybe use pid()/tid()/core() names here... they are accessors: https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
It says "accessors and mutators *may* be named like variables". I don't know how to interpret that, but the rest of the module uses "Get..." so I would stick with it.